src/app/controllers/albumCtrl.js

```diff
@@ -0,0 +1,41 @@
+export function albumCtrl($scope, apiService, $routeParams, favoriteServ, storageServ) {
+    $scope.albumCtrl = this;
+    this.trackPlaying = new Audio("");
+
+	this.getAlbum = function(){   		
+	        apiService.getAlbum($scope.albumCtrl.currentAlbum.id)
+	        .then(function successCallback(response) {
+	        	$scope.albumCtrl.currentAlbum = response.data;
+	        	$scope.albumCtrl.currentAlbum.year = parseInt(response.data.release_date.substring(0, 4));
+			}, function errorCallback(response) {
+				console.log(response);
+			});
+    };
+
+    this.tracksNotEmpty = function() {
+    	return $scope.albumCtrl.currentAlbum.tracks.items.length > 0;
+    };
+
+    this.playTrack = function(url) {
```

Comment:
you can bind(this) to those functions to avoid doing repeated reference to $scope.albumCtrl, could you check it?